reporter/jsonConsole.go

```diff
@@ -84,9 +89,12 @@ type executionError struct {
 	Text       string `json:"text"`
 	Filename   string `json:"filename"`
 	Message    string `json:"message"`
+	LineNo     string `json:"lineNo"`
 	StackTrace string `json:"stackTrace"`
 }
 
+var stepCache = make(map[*gm.ScenarioInfo][]*stepInfo)
```

Comment:
This cache can be added to JSONConsole instead of a global var.

Reply:
In that case, the private methods who are using the cache has to be bound with JSONConsole

Reply:
Or it can be passed as param.